lib/rgeo/geos.rb

```diff
@@ -26,7 +26,11 @@ module Geos
     begin
       require_relative "geos/geos_c_impl"
     rescue LoadError
-      # continue
+      begin
+        require "geos/geos_c_impl"
+      rescue LoadError
+        # continue
+      end
```

Comment:
Can you just add a little comment here explaining that we fall back to a system-wide search if the local search fails?

Reply:
Of course, inline comment or do you have some convention, e.g. to oput it above the block?

Reply:
Sorry, where you put it is great thank you